src/auth/bearer/fake/fake.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2014, Treehouse Networks Ltd. New Zealand
```

Comment:
Oh. Understood now (I think).

Treehouse should have had a CONTRIBUTORS entry since long ago for the other fake helpers. For the same reason. Adding it now, and the missing CREDITS entry too.